invenio.cfg

```
@@ -302,7 +302,10 @@ OAUTHCLIENT_REMOTE_APPS = {
     "github": github_remote_app,
     "openaire_aai": REMOTE_SANDBOX_APP,
 }
-
+OAUTHCLIENT_REST_REMOTE_APPS = dict(
+    github=github_remote_app,
```

Just for consistency as above:
```suggestion
    "github": github_remote_app,
```